src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -194,7 +194,9 @@ public bool IsCurrentInstance
 			}
 		}
 
-		public PaneHolderPage()
+        public readonly VirtualKey PlusKey = (VirtualKey)187;
```

Comment:
I suppose the variable here is used to be able to bind it in the .xaml file. But would it be possible to use a simple key, like in l.181 of `PaneHolderPage.xaml`?

Reply:
The Key property must be a VirtualKey, so again it doesn't support the plus key directly. I don't know if there is a cleaner workaround.

Reply:
Not that I can think of sadly.